src/components/Text/Base.styles.js

```diff
@@ -2,14 +2,24 @@ import styled from "styled-components";
 
 import { typography } from "../../theme";
 import { getFontColor } from "../../utils/typography";
+import { mediumAndUp, smallAndUp } from "../../theme/mediaQueries";
 
 const Base = styled.div`
-  font-size: ${({ size }) => typography.size[size] || typography.size.hecto};
+  font-size: ${({ size }) => typography.size[size.small]};
   font-weight: ${({ weight }) =>
-    typography.weight[weight] || typography.size.regular};
+    typography.weight[weight] || typography.weight.regular};
```

Comment:
You don't need to fallback on `"regular"` since it is the default value for the `weight` prop